src/components/promotionspopup.tsx

```diff
@@ -18,7 +18,6 @@ const PromotionsPopup = () => {
     };
     try {
       const data = await fetchData(URL_PROMOTIONS, params);
```

Comment:
are u not getting any eslint warning for `data` being unused ?

Reply:
It shows that the 'data' variable is declared, but its value is never read. We might need this variable in the future when integrating the API, which is why I've kept it there